src/Components/Main/Main.tsx

```diff
@@ -0,0 +1,31 @@
+import { useContext, useMemo } from 'react';
+
+import { TodoItem } from '../TodoItem';
+
+import { Context } from '../../Context';
+import { Filter } from '../../types/Filter';
+
+export const Main = () => {
```

Comment:
```suggestion
export const TodoList = () => {
```
it'll better describe what exactly it is